esx_service/utils/kvESX.py

```diff
@@ -31,6 +31,7 @@
 
 # VSphere lib to access ESX proprietary APIs.
 DISK_LIB = "/lib/libvmsnapshot.so"
+DISK_LIB64 = "/lib64/libvmsnapshot.so"
```

Comment:
I am still not sure how would it help for esx_service to run. Don't we need to build our vmci lib in 32 and 64 and load the correct version? And do you plan to figure out what is the correct version based on try/except above ? And how it is tested? 
All in all, I think it is better to merge the syntax repair code (print ->print() ) and the likes - with related pycompile test. 
```python
import py_compile
py_compile.compile('my_script.py')
```

And then have a separate PR to build our vmci libs in 64, and load based on version of python or version of ESX - - with related tests on python 3.5